dsgrid/config/annual_time_dimension_config.py

```diff
@@ -70,73 +72,22 @@ def convert_dataframe(
         self,
         df,
         project_time_dim,
-        model_years=None,
-        value_columns=None,
+        value_columns: set[str],
+        scratch_dir_context: ScratchDirContext,
         wrap_time_allowed=False,
         data_adjustment=None,
     ):
-        assert model_years is not None
-        assert value_columns is not None
-        match self.model.measurement_type:
-            case MeasurementType.MEASURED:
```

Comment:
I am removing support for measured historical time. What we were doing is not correct because there is no definition of the interval being measured. The measured/min/max/mean types need to specify the interval for the measurement.

Reply:
Annual time tells you the interval--one year. But looking at the options, I see that "measured" is essentially synonymous with "instantaneous", i.e., more like power than energy units, so I'm fine with removing support for now.